tools/webtransport/h3/webtransport_h3_server.py

```diff
@@ -44,46 +45,49 @@
 quic_connection_logger.setLevel(logging.WARNING)
 
 
-class H3ConnectionWithDatagram04(H3Connection):
+class H3DatagramSetting(IntEnum):
+    # https://datatracker.ietf.org/doc/html/draft-ietf-masque-h3-datagram-04#section-8.1
+    DRAFT04 = 0xffd277
+    # https://datatracker.ietf.org/doc/html/rfc9297#name-http-3-setting
+    RFC = 0x33
+
+
+class H3ConnectionWithDatagram(H3Connection):
     """
     A H3Connection subclass, to make it work with the latest
     HTTP Datagram protocol.
     """
-    H3_DATAGRAM_04 = 0xffd277
-    # https://datatracker.ietf.org/doc/html/draft-ietf-httpbis-h3-websockets-00#section-5
+    # https://datatracker.ietf.org/doc/html/rfc9220#name-iana-considerations
```

Comment:
nit: Should this be https://datatracker.ietf.org/doc/html/rfc9220#section-5-2.2.1 to be more specific?